nemo/collections/cv/models/mnist_lenet5.py

```diff
@@ -100,9 +96,6 @@ def __init__(self, cfg: MNISTLeNet5Config = MNISTLeNet5Config()):
         # This will be set by setup_test_data
         self._test_dl = None
 
```

Comment:
There's no need for these three _dl lines anymore.

Reply:
removed